src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -45,9 +45,9 @@
 import java.util.Map;
 
 @TruffleLanguage.Registration(
-        name = RubyLanguage.NAME,
-        id = RubyLanguage.ID,
-        version = RubyLanguage.RUBY_VERSION,
+        name = "ruby",
```

Comment:
Is named changed on purpose from `Ruby` to `ruby`?
I think it should be `Ruby`.
`id` is already `ruby`.

Reply:
Fixed.